gpustack/policies/candidate_selectors/base_candidate_selector.py

```diff
@@ -888,13 +888,14 @@ async def _validate_distributed_limit_per_worker(self, worker: Worker) -> bool:
 
     def _check_gpu_cnt_divisibility(self, selected_gpu_cnt: Optional[int] = None):
         """
-        Check whether vLLM's constraint on parameter divisibility is satisfied.
+        Check whether InferenceBackend's constraint of parameter divisibility is satisfied.
         1. num_attention_heads
         2. vocab_size
         """
         if not selected_gpu_cnt:
             selected_gpu_cnt = find_int_parameter(
-                self._model.backend_parameters, ["tensor-parallel-size", "tp"]
+                self._model.backend_parameters,
+                ["tensor-parallel-size", "tp", "tp-size"],
```

Comment:
Parameter names are specific to inference backends so we should not have these in base_selector. If we need tp_size, can it be passed via function argument?